test/extended/prometheus/prometheus.go

```diff
@@ -438,7 +430,7 @@ var _ = g.Describe("[sig-instrumentation] Prometheus [apigroup:image.openshift.i
 					targets.Expect(labels{"job": "node-exporter"}, "up", "^https://.*/metrics$"),
 					targets.Expect(labels{"job": "prometheus-operator"}, "up", "^https://.*/metrics$"),
 					targets.Expect(labels{"job": "alertmanager-main"}, "up", "^https://.*/metrics$"),
-					targets.Expect(labels{"job": "crio"}, "up", "^http://.*/metrics$"),
+					targets.Expect(labels{"job": "crio"}, "up", "^https://.*/metrics$"),
```

Comment:
We might have  a chicken-and-egg problem?
```suggestion
					targets.Expect(labels{"job": "crio"}, "up", "^http(s)?://.*/metrics$"),
```

Reply:
@simonpasquier Jobs look better... Ok to remove the hold?